index.js

```diff
@@ -0,0 +1,29 @@
+function calculatetips(){
+    // document.getElementById("tips").innerHTML = "hello"
+var amountinput = document.getElementById("billamount").value;
+if(amountinput== 0 || amountinput==" "){
+     var a = prompt("enter a value for Bill amount:")
+};
+//if user forget to enter the bill amount
+document.getElementById("billamount").value = a;
+var dropdown = document.getElementById("dropdown").value;
+
+var numberpeople = document.getElementById("numberpeople").value;
+
+if(dropdown == 0  && numberpeople == " "){
+    alert("enter good value");
+}
+var total = (amountinput * dropdown)/numberpeople;
```

Comment:
you might want to convert numberpeople to int using parseInt as your input type is text or just change the input type to number.